src/styles/_playground.scss

```diff
@@ -7,6 +7,6 @@
 
 .kineticjs-content {
   width: 100% !important;
-  height: 40vh !important; // 100% results in 0 height, 100vh is too big.
-  overflow: scroll;
+  // height: 40vh !important; // 100% results in 0 height, 100vh is too big.
+  // overflow: scroll;
```

Comment:
Don't comment out code unless there's really some need to? (e.g. there are plans to reimplement this)

Reply:
I'm not entirely sure whether those attributes will be needed again in the future, but I'll delete them for now.